forms/src/pages/SurveyCreationPage/SurveyCreationPage.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from "react";
+import SurveySection from "./components/form/SurveySection/SurveySection";
+import SurveyTitle from "./components/form/SurveyTitle/SurveyTitle";
+import "./SurveyCreationPage.scss";
+import plus from "../../assets/plus.svg";
+import { DragDropContext, Droppable, Draggable } from "react-beautiful-dnd";
+import { iQuestion, QuestionType } from "../../interfaces/iQuestion";
+import { sectionsContext } from "../../context/sectionsContext";
+
+function SurveyCreationPage(props: { surveyName: string }) {
```

Comment:
In all of your components, I'd suggest destructuring props in the function signature so it's easy to see what props are being passed and whether they're in use. Also makes it easier to refer to props in your components without needing to prefix with `props.` every time.

```suggestion
function SurveyCreationPage({surveyName}: { surveyName: string }) {
```